Jenkinsfile

```diff
@@ -27,41 +27,45 @@ def getChangeDetail() {
     return changeDetail
 }
 
-def sendEmailNotification() {    
-
-    script{ 
-        final def EXTRECIPIENTS = emailextrecipients([
-            [$class: 'CulpritsRecipientProvider'],
-            [$class: 'RequesterRecipientProvider'],
-            [$class: 'DevelopersRecipientProvider']
-        ])
+def IS_MASTER_BRANCH = env.BRANCH_NAME == "master"
+def IS_RELEASE_BRANCH = (env.BRANCH_NAME ==~ /\d+\.\d+\.\d+/)
 
-        def RECIPIENTS = EXTRECIPIENTS != null ? EXTRECIPIENTS : env.CHANGE_AUTHOR_EMAIL;
-        def SUBJECT = "${env.JOB_NAME} - Build #${env.BUILD_NUMBER} - ${currentBuild.currentResult}!"
-        def CHANGE = getChangeDetail();
-        
-        emailext(
-            to: RECIPIENTS,
-            subject: "${SUBJECT}",
-            body: """
-                <b>Build result:</b> 
-                <br><br>
-                ${currentBuild.currentResult}
-                <br><br>
-                <b>Project name - Build number:</b>
-                <br><br>
-                ${currentBuild.fullProjectName} - Build #${env.BUILD_NUMBER}
-                <br><br>
-                <b>Check console output for more detail:</b> 
-                <br><br>
-                <a href="${currentBuild.absoluteUrl}console">${currentBuild.absoluteUrl}console</a>
-                <br><br>
-                <b>Changes:</b> 
-                <br><br>
-                ${CHANGE}
-                <br><br>
-            """
-        );
+def sendEmailNotification() {    
+    script { 
```

Comment:
I don't think you can put a `script` here since we're already in regular groovy, I think jsut remove it (and the closing brace)

Reply:
Good catch. Removed.